lldb/test/API/lang/swift/cxx_interop/forward/verbose_trap/TestSwiftForwardInteropVerboseTrap.py

```diff
@@ -0,0 +1,22 @@
+
+"""
+Test that verbose_trap works on forward interop mode.
+"""
+from lldbsuite.test.lldbtest import *
+from lldbsuite.test.decorators import *
+
+
+class TestSwiftForwardInteropVerboseTrap(TestBase):
+
+    @swiftTest
+    def test(self):
+        self.build()
```

Comment:
Why not just  do:
```
lldbutil.run_to_source_breakpoint(self,  "Break here", lldb.SBFileSpec("main.swift")
```
?

Reply:
Is it because we don't expect tp hit the breakpoint?

Reply:
Yup, `lldbutil.run_to_source_breakpoint` asserts that we stopped at some breakpoint. Which won't happen for this test.

Might be worth creating a wrapper for the "just launch the executable" case. That way we could make use of `__buitin_debugtrap` in more tests too.